AsyncAwait.Task2.CodeReviewChallenge/Services/PrivacyDataService.cs

```diff
@@ -4,9 +4,9 @@ namespace AsyncAwait.Task2.CodeReviewChallenge.Services;
 
 public class PrivacyDataService : IPrivacyDataService
 {
-    public Task<string> GetPrivacyDataAsync()
+    public async ValueTask<string> GetPrivacyDataAsync()
     {
-        return new ValueTask<string>("This Policy describes how async/await processes your personal data," +
-                                     "but it may not address all possible data processing scenarios.").AsTask();
+        return await ValueTask.FromResult("This Policy describes how async/await processes your personal data," +
```

Comment:
Do you need `await` if you return result?

Reply:
Oh, I guess we have to keep GetPrivacyData as async because of task purposes. My original question was should we await FromResult operation? Because keep in mind that every `await` adds overhead.